packages/components/organisms/f-header/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v10.15.1
```

Comment:
Nit: this is probably more of a minor version than a patch, and can you also use "November" instead of "Nov" while you're at it 😄